cloud/aws-functions/orphaned_eks_clusters.py

```diff
@@ -58,29 +58,51 @@ def wait_for_node_group_delete(autoscaling_client, autoscaling_group_name):
     else:
         logging.error(f"Node group {autoscaling_group_name} was not deleted in {timeout} seconds.")
 
-def delete_nodegroup(aws_region, cluster_name):
+def delete_nodegroup(aws_region):
     autoscaling_client = boto3.client('autoscaling', region_name=aws_region)
-    ec2 = boto3.resource('ec2')
-
-    autoscaling_group_name = ""
-    for instance in ec2.instances.all():
-        tags = instance.tags
-        tags_dict = {item['Key']: item['Value'] for item in tags}
-        if cluster_name not in tags_dict['Name']:
-            continue
-        state = instance.state['Name']
-        if tags_dict['alpha.eksctl.io/cluster-name'] and tags_dict[
-            'alpha.eksctl.io/cluster-name'] == cluster_name and state == 'running':
-            autoscaling_group_name = tags_dict['aws:autoscaling:groupName']
-            break
-        else:
-            continue
-    if len(autoscaling_group_name) != 0:
-        try:
-            autoscaling_client.delete_auto_scaling_group(AutoScalingGroupName=autoscaling_group_name, ForceDelete=True)
-            wait_for_node_group_delete(autoscaling_client, autoscaling_group_name)
-        except Boto3Error as e:
-            logging.error(f"Deleting autoscaling group {autoscaling_group_name} failed with error: {e}")
+    auto_scaling_groups = []
+
+    paginator = autoscaling_client.get_paginator('describe_auto_scaling_groups')
+    for page in paginator.paginate():
+        print(f'page {page}')
+        for group in page['AutoScalingGroups']:
+            print(f'group {group}')
+            tags = group.get('Tags', [])
+            print(f'tags {tags}')
+            team_value = False
+            delete_cluster = False
+
+            for tag in tags:
+                # print(f'tag {tag}')
+                # print(f"tagKey {tag['Key']}")
+                # print(f"tagValue {tag['Value']}")
```

Comment:
If we are just printing above and not here perhaps delete?

Reply:
I will delete it) sorry